mmv1/third_party/terraform/services/datastream/resource_datastream_connection_profile_test.go

```diff
@@ -247,3 +346,117 @@ resource "google_datastream_connection_profile" "mysql_con_profile" {
+	context["lifecycle_block"] = ""
+	if preventDestroy {
+		context["lifecycle_block"] = `
+        lifecycle {
+            prevent_destroy = true
+        }`
+	}
+	context["private_key"] = private_key
+	context["public_key"] = public_key
+
+	return acctest.Nprintf(`
+resource "google_sql_database_instance" "instance" {
+        name             = "tf-test-my-database-instance%{random_suffix}"
+        database_version = "POSTGRES_14"
+        region           = "us-central1"
+        settings {
+            tier = "db-f1-micro"
+            ip_configuration {
+    
+            // Datastream IPs will vary by region.
+            authorized_networks {
+                value = "34.71.242.81"
+            }
+    
+            authorized_networks {
+                value = "34.72.28.29"
+            }
+    
+            authorized_networks {
+                value = "34.67.6.157"
+            }
+    
+            authorized_networks {
+                value = "34.67.234.134"
+            }
+    
+            authorized_networks {
+                value = "34.72.239.218"
+            }
+        }
+    }
+    
+        deletion_protection  = "false"
+    }
+    
+resource "google_sql_database" "db" {
+        instance = google_sql_database_instance.instance.name
+        name     = "db"
+    }
+ 
+resource "google_sql_user" "user" {
+        name = "user"
+        instance = google_sql_database_instance.instance.name
+        password = "Ckrw75FbtmKrTKCtWPFJS54cTdbGC8D82rJwp3gV"
+    }
+
+resource "google_compute_instance" "default" {
+		name         = "test-bastion"
+		machine_type = "e2-small"
+		zone         = "us-central1-a"
+		boot_disk {
+			initialize_params {
+			image = "debian-11-bullseye-v20241009"
+		  }
+		}
+
+		network_interface {
+		  network = "default"
+		  access_config {
+
+		  }
+		}
+		metadata = {
+		  "ssh-keys" = "${google_sql_user.user.name}:%{public_key}"
+		}
+	  }
+  
+resource "google_compute_firewall" "ssh" {
+		name = "tf-test-%{random_suffix}"
+		allow {
+		  ports    = ["22"]
+		  protocol = "tcp"
+		}
+		direction     = "INGRESS"
+		network       = "default"
+		priority      = 1000
+		source_ranges = ["0.0.0.0/0"]
+		target_tags   = ["ssh"]
+}
+
+resource "google_datastream_connection_profile" "ssh_connectivity_profile" {
+        display_name          = "Source connection profile"
+        location              = "us-central1"
+        connection_profile_id = "tf-test-mysql-profile%{random_suffix}"
+
+    	postgresql_profile {
+        	hostname = google_sql_database_instance.instance.public_ip_address
+        	username = google_sql_user.user.name
+        	password = google_sql_user.user.password
+        	database = google_sql_database.db.name
+    	}
+
+    	forward_ssh_connectivity {
+        	hostname = google_compute_instance.default.network_interface.0.access_config.0.nat_ip
+        	username = google_sql_user.user.name
+        	port     = 5432
+        	private_key = "%{private_key}"
```

Comment:
This is failing with `Error: Invalid multi-line string`. I think you can change it to something like:

```
private_key = <<EOT
%{private_key}
EOT
```

Reply:
thanks, will do!